carbon-design-system/carbon
# feat(colors): add support for Sass Modules

Adds support for Sass Modules with the colors package through `@carbon/colors/scss/module.scss` 

The generated code for this module looks like:

![Screen Shot 2020-12-03 at 15 54 42](https://user-images.githubusercontent.com/3901764/101094948-ce69e880-3582-11eb-9445-798f7eee2019.png)

This PR also adds in an example for bringing in `@carbon/colors` to a project and using it as a Sass Module 👀 


#### Changelog

**New**

- Add `module.scss` to `@carbon/colors`

**Changed**

**Removed**

#### Testing / Reviewing

- Pull down the PR
- Run `yarn build` in `@carbon/colors`
- Verify `module.scss` file is built in `packages/colors/scss/module.scss`
- Run the example in `colors/examples/sass-modules` and then run `yarn install` and `yarn develop` and verify the color variables work as intended
